compiler/passes/errorHandling.cpp

```diff
@@ -65,7 +107,12 @@ ErrorHandlingVisitor::ErrorHandlingVisitor(ArgSymbol*   _outError,
 bool ErrorHandlingVisitor::enterTryStmt(TryStmt* node) {
   SET_LINENO(node);
 
-  VarSymbol*   errorVar     = newTemp("error", dtObject);
+  // TODO: nested try
+  if (!tryStack.empty()) {
+    USR_FATAL("nested try is not yet supported");
+  }
```

Comment:
I am curious, do you think you can get nested try working soon? I think we have the right framework here... consider it a challenge :)